Makefile

```
@@ -110,4 +113,4 @@ tfgen: install_plugins
 bin/pulumi-java-gen: 
 	$(shell pulumictl download-binary -n pulumi-language-java -v $(JAVA_GEN_VERSION) -r pulumi/pulumi-java)
 
-.PHONY: development build build_sdks install_go_sdk install_java_sdk install_python_sdk install_sdks only_build build_dotnet build_go build_java build_nodejs build_python clean cleanup help install_dotnet_sdk install_nodejs_sdk install_plugins lint_provider provider test tfgen
\ No newline at end of file
+.PHONY: development build build_sdks examples install_go_sdk install_java_sdk install_python_sdk install_sdks only_build build_dotnet build_go build_java build_nodejs build_python clean cleanup help install_dotnet_sdk install_nodejs_sdk install_plugins lint_provider provider test tfgen
```

Just so diffs become meaningful (assuming we are now editing this file by hand.
```suggestion
.PHONY: development build build_sdks \
        examples install_go_sdk install_java_sdk \
        install_python_sdk install_sdks only_build \
        build_dotnet build_go build_java \
        build_nodejs build_python clean \
        cleanup help install_dotnet_sdk \
        install_nodejs_sdk install_plugins lint_provider \
        test tfgen provider
```